src/edt/edt/edtPCellParametersPage.cc

```diff
@@ -762,6 +775,11 @@ PCellParametersPage::get_parameters_internal (db::ParameterStates &states, bool
               ps.set_value (tl::Variant (v));
               lay::indicate_error (le, (tl::Exception *) 0);
 
+              if (p->get_range().has_value())
```

Comment:
Checking the value here does not have the effect of rejecting the wrong value as "ps.set_value" already happened.

Reply:
The gui will reject in that case. But will move to above ps.set_value...